hiedb.el

```diff
@@ -52,74 +52,64 @@
   "Query hiedb for references of symbol at point."
   (interactive)
   (let ((module (hiedb-module-from-path)))
-    (hiedb-query-point-refs module (line-number-at-pos) (1+ (current-column)))
-    (switch-to-buffer-other-window "*hiedb*")
-    (special-mode)))
+    (hiedb-query-point-refs module (line-number-at-pos) (1+ (current-column)))))
```

Comment:
Given `hiedb-query-xxx` is proxy to `call-hiedb`, I'm wondering to get rid of them.